pkgs/applications/networking/cluster/k3sup/default.nix

```diff
@@ -0,0 +1,58 @@
+{ lib
+, buildGoModule
+, fetchFromGitHub
+, makeWrapper
+, bash
+, openssh
+}:
+
+buildGoModule rec {
+  pname = "k3sup";
+  version = "0.11.3";
+
+  src = fetchFromGitHub {
+    owner = "alexellis";
+    repo = "k3sup";
+    rev = version;
+    sha256 = "sha256-6WYUmC2uVHFGLsfkA2EUOWmmo1dSKJzI4MEdRnlLgYY=";
+  };
+
+  buildInputs = [ makeWrapper ];
+
+  overrideModAttrs = (_: {
+    postBuild = ''
+      substituteInPlace vendor/github.com/alexellis/go-execute/pkg/v1/exec.go \
+        --replace "/bin/bash" "bash"
+    '';
+  });
+
+  vendorSha256 = "sha256-VT1Xz1FOPy5fW7p4qskYJzmS7xaXt4mlMLkoHpzHMq0=";
+
+  CGO_ENABLED = 0;
+
+  ldflags = [
+    "-s" "-w"
+    "-X github.com/alexellis/k3sup/cmd.GitCommit=ref/tags/${version}"
+    "-X github.com/alexellis/k3sup/cmd.Version=${version}"
+  ];
+
+  postInstall = ''
+    wrapProgram "$out/bin/k3sup" \
+      --prefix PATH : ${lib.makeBinPath [ openssh bash ]}
+  '';
+
+  meta = with lib; {
+    homepage = "https://github.com/alexellis/k3sup";
+    description = "Bootstrap Kubernetes with k3s over SSH";
+    license = licenses.mit;
+    maintainers = with maintainers; [ welteki ];
+    platforms = [
```

Comment:
My understanding is that `platforms.linux ++ platforms.darwin` is the default if platforms is not defined, so platforms doesn't need to be set (per https://github.com/NixOS/nixpkgs/pull/153464). Or is it that the arm6/7 arch's not included by default?

Reply:
Arm6/7 are included by default but I'm explicitly setting the platforms supported by upstream. Already discussed this once here: https://github.com/NixOS/nixpkgs/pull/145080#issuecomment-967145799

Reply:
We don't want platforms set unnecessarily as it end up being cargo culted in other derivations, please remove them. They should be removed from `arkade` and `fass-cli` as well but I can to that in another PR.

Reply:
Removed them as requested. Not entirely clear to me, should I open another PR to remove them from `arkade` and `faas-cli` or will you do that?